lighthouse-core/audits/dobetterweb/js-libraries.js

```diff
@@ -44,6 +44,8 @@ class JsLibrariesAudit extends Audit {
   static audit(artifacts) {
     const libDetails = artifacts.Stacks
       .filter(stack => stack.detector === 'js')
+      // Don't show the fast paths in the table. `react-path` is a typo in js-library-detector.
+      .filter(stack => !stack.id.endsWith('-fast') && stack.id !== 'react-path')
```

Comment:
bummer, we should probably file a PR upstream for that...

Reply:
it was fixed. https://github.com/johnmichel/Library-Detector-for-Chrome/commit/0fd3b90467b2e49dffac69bdc764900be44230eb

@housseindjirdeh could you do a new release for this pkg?